components/Create.js

```diff
@@ -549,3 +595,100 @@ function CertTemp3Inputs({
         </>
     );
 }
+
+function CertTemp4Inputs({
+    setTitle,
+    title,
+    setLine1,
+    line1,
+    setSignature,
+    signature,
+    issueDate,
+    setIssueDate,
+    setLeadUniversity,
+    leadUniversity,
+    setDate,
+    date,
+    setNames,
+    names,
+    disabled,
+    handleCreateBtn,
+}) {
+    return (
+        <>
+            <input
+                onChange={(e) => {
+                    setTitle(e.target.value);
+                }}
+                value={title}
+                className={styles.input}
+                placeholder={`Title`}
+            />
+            <textarea
+                onChange={(e) => {
+                    setLine1(e.target.value);
+                }}
+                value={line1}
+                className={styles.textBox}
+                placeholder="Line 1"
+                style={{
+                    minHeight: "100px",
+                }}
+            />
+            <textarea
+                onChange={(e) => {
+                    setLeadUniversity(e.target.value);
+                }}
+                value={leadUniversity}
+                className={styles.textBox}
+                placeholder="Lead Name, University"
+                style={{
+                    minHeight: "100px",
+                }}
+            />
+            <input
+                onChange={(e) => {
+                    setDate(e.target.value);
+                }}
+                value={date}
+                className={styles.input}
+                placeholder={new Date().toLocaleDateString("en-US", {
+                    year: "numeric",
+                    month: "short",
+                    day: "numeric",
+                })}
+            />
+            <input
+                onChange={(e) => {
+                    setIssueDate(e.target.value);
+                }}
+                value={issueDate}
+                className={styles.input}
+                placeholder={new Date().toLocaleDateString("en-US", {
+                    year: "numeric",
+                    month: "short",
+                    day: "numeric",
+                })}
+            />
```

Comment:
What is the difference between issueDate and date?

You are not using the issueDate with the certificate template and you haven't modified helper function responsible for sending the data to the firebase database to include the value of issueDate.

I can't seem to find any use for that variable